pkg/virt-api/rest/subresource.go

```diff
@@ -173,10 +173,6 @@ func (app *SubresourceAPIApp) VNCRequestHandler(request *restful.Request, respon
 			log.Log.Object(vmi).Reason(err).Error("Can't establish VNC connection.")
 			return errors.NewBadRequest(err.Error())
 		}
-		condManager := controller.NewVirtualMachineInstanceConditionManager()
```

Comment:
Do we want also VNC connections to establish for paused VMs? What's the use case for that?

Reply:
User can choose between serial and VNC console. So we want both.